### PR TITLE
Hotfix/work page

### PR DESCRIPTION
**Pull request by Alesyaa103**

### CODE REVIEW

File: client/src/components/AccountManager/index.tsx [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
-	const DeleteUser = () => {
+	const [isDelete, setIsDelete] = useState(false);
+
+	const onClose = () => {
+		setIsDelete(false);
+	};
+
+	const showDeleteModal = () => {
+		setIsDelete(true);
+	};
```
Comment (YrPryhoda) [inline client/src/components/AccountManager/index.tsx:24]: you may change this 2 functions with 1
const toggleModal = () => setIsDelete(!isDelete)

File: client/src/components/DeleteUserModal/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import { Button, Modal } from 'semantic-ui-react';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+	onClose: () => void;
+	deleteUser: () => void;
+}
+
+const DeleteUserModal: React.FC<Props> = (props) => {
+	const { deleteUser, onClose } = props;
```
Comment (YrPryhoda) [inline client/src/components/DeleteUserModal/index.tsx:11]: you can destructure arguments straight in func declare
const DeleteUserModal = ({deleteUser, onClose}:props) => ...

Comment (Alesyaa103) [reply client/src/components/DeleteUserModal/index.tsx:11]: Roman asked destructure props in function body

File: client/src/components/DeleteUserModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	const { t } = useTranslation();
+
+	const submitForm = () => {
+		deleteUser();
+		onClose();
+	};
+
+	return (
+		<Modal closeIcon onClose={() => onClose()} open={true} size={'tiny'} dimmer="inverted">
```
Comment (YrPryhoda) [inline client/src/components/DeleteUserModal/index.tsx:20]: if you call func without arguments, you can type it without excess adition arrow func

onClose={onClose}

File: client/src/components/ProjectCard/index.tsx [hunk 4]
```diff
@@ -19,10 +25,10 @@ const ProjectCard: React.FC<Props> = (props: Props) => {
 			</div>
 			<div className={styles.issues}>
 				<p className={styles.text}>
-					My open issues <span className={styles.count}>0</span>
+					My open issues <span className={styles.count}>{undoneIssues}</span>
 				</p>
 				<p className={styles.text}>
-					My done issues <span className={styles.count}>0</span>
+					My done issues <span className={styles.count}>{doneIssues}</span>
```
Comment (YrPryhoda) [inline client/src/components/ProjectCard/index.tsx:31]: Add please localization

File: client/src/components/WorkAssignedBlock/index.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+	>([]);
+	const getIssues = async (id: string): Promise<void> => {
+		const issues = await requestGetUserIssues(id);
+		setIssues(issues);
+	};
+	useEffect(() => {
+		getIssues(id);
+	}, [id]);
+
+	console.log(issues);
```
Comment (YrPryhoda) [inline client/src/components/WorkAssignedBlock/index.tsx:21]: useless console.log

File: client/src/containers/ProfilePage/index.tsx [hunk 6]
```diff
@@ -15,14 +14,23 @@ import { NotificationManager } from 'react-notifications';
...
 		userData: state.user,
 		currentUser: state.auth.user,
 	}));
+	const { projects, teammates, teams } = useSelector((state: RootState) => ({
+		projects: state.projects.projects,
+		teammates: state.peoplePage.people,
+		teams: state.peoplePage.teams,
+	}));
+
+	const [data, setData] = useState({
```
Comment (YrPryhoda) [inline client/src/containers/ProfilePage/index.tsx:29]: can you rename variable "data" to something more exact

File: client/src/containers/WorkPage/logic/saga.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+import { UserActivityState } from './state';
+import { NotificationManager } from 'react-notifications';
+
+function* getIssue(action: ReturnType<typeof actions.requestGetAssignedIssues>) {
+	const { id } = action;
+	try {
+		const issues: Partial<UserActivityState> = yield call(requestGetUserIssues, id);
+		yield put(actions.updateData({ partialState: issues }));
+	} catch (error) {
+		NotificationManager.error('Could not load data', 'Error', 4000);
```
Comment (YrPryhoda) [inline client/src/containers/WorkPage/logic/saga.ts:14]: add, please, localization (example TeamPage>logic>saga)

File: server/src/repositories/user.repository.ts [hunk 8]
```diff
@@ -94,8 +114,8 @@ export class UserRepository extends Repository<UserProfile> {
 			.leftJoinAndSelect('user.teamsOwner', 'team')
 			.getOne();
 		const teamRepository = getCustomRepository(TeamRepository);
-		user?.teamsOwner?.map((item) => {
-			teamRepository.deleteOneById(item.id);
+		user?.teamsOwner?.map(async (item) => {
```
Comment (YrPryhoda) [inline server/src/repositories/user.repository.ts:117]: change map to forEach:  Map creates a new array

**Review by YrPryhoda — CHANGES_REQUESTED** [review]

File: client/src/components/DeleteUserModal/index.tsx (hunk 2 above)
Comment (YrPryhoda) [reply client/src/components/DeleteUserModal/index.tsx:11]: ok )

**Review by YrPryhoda — APPROVED** [review]

**Pull request merged**